scalameta/dialects/shared/src/main/scala/scala/meta/dialects/package.scala

```diff
@@ -147,15 +147,12 @@ package object dialects {
   )
 
   implicit val Scala212 = Scala211.copy(
-    allowLiteralTypes = false, // Scheduled to be included in 2.12.2
```

Comment:
This is unrelated to this PR, we can merge this change immediately if you open a separate PR @dwijnand :)